src/main/java/org/spine/iquestionapi/ShellCommands.java

```diff
@@ -0,0 +1,83 @@
+package org.spine.iquestionapi;
+
+import java.util.List;
+
+import org.apache.commons.lang3.EnumUtils;
+import org.spine.iquestionapi.model.EmailDomain;
+import org.spine.iquestionapi.model.User;
+import org.spine.iquestionapi.model.User.Role;
+import org.spine.iquestionapi.repository.EmailDomainRepo;
+import org.spine.iquestionapi.repository.UserRepo;
+import org.spine.iquestionapi.security.JWTUtil;
+import org.spine.iquestionapi.util.StringUtil;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.shell.standard.ShellComponent;
+import org.springframework.shell.standard.ShellMethod;
+
+@ShellComponent
+public class ShellCommands {
+
+    @Autowired private EmailDomainRepo emailDomainRepo;
+    @Autowired private UserRepo userRepo;
+    @Autowired private PasswordEncoder passwordEncoder;
+    @Autowired private JWTUtil jwtUtil;
+
+    @ShellMethod("Add a user")
+    public String addUser(String name, String email, String organization, String role, String password) {
+        // check if all the fields are filled
+        if (name.isEmpty() || email.isEmpty() || organization.isEmpty() || role == null || password.isEmpty()) {
+            return "Please fill all the fields";
+        }
+        
+        // check if role is a valid role
+        if (!EnumUtils.isValidEnum(Role.class, role.toUpperCase())) {
+            return "Role is not valid, use one of the following: " + Role.values().toString().toLowerCase();
+        }
+
+        // check if the email domain is allowed, if not return
+        List<EmailDomain> emailDomains = emailDomainRepo.findAll();
+
+        // Check if email is of a valid type
+        if (!StringUtil.isValidEmail(email, emailDomains)) {
+            return "Email domain not allowed, you can add it via the addEmailDomain command";
```

Comment:
Voor alle returnes, English code, Dutch speech

Reply:
I want to avoid non-english speech in the backend application. This'll only get used by experts (who might be outsourced) so I'd prefer to keep this in english.